include/picongpu/param/physicalConstants.param

```diff
@@ -46,18 +46,22 @@ namespace picongpu
 
         // Electron properties
         /** unit: kg */
-        constexpr float_64 ELECTRON_MASS_SI = 9.109382e-31;
+        constexpr float_64 ELECTRON_MASS_SI = 9.1093837015e-31;
         /** unit: C */
         constexpr float_64 ELECTRON_CHARGE_SI = -1.602176e-19;
 
-        /* atomic unit for energy:
-         * 1 Rydberg = 27.21 eV --> converted to Joule
+        /** atomic unit for energy:
+         * 2 Rydberg = 27.21 eV --> converted to Joule
          */
         constexpr float_64 ATOMIC_UNIT_ENERGY = 4.36e-18;
```

Comment:
Why is this two digits of precision while everything else has so high precision?

Reply:
This value is used by the existing ionization implementation by marco, I would guess it was deemed good enough by him.

I am hesitant to touch them since I haven't looked in detail into the normalization of of marcos implementation, if we want to change them i would propose to do that in a separate PR.

Reply:
I agree with @BrianMarre